src/core/interp.c

```diff
@@ -355,9 +355,6 @@ void MVM_interp_run(MVMThreadContext *tc, void (*initial_invoke)(MVMThreadContex
                 GC_SYNC_POINT(tc);
                 goto NEXT;
             }
-            OP(DEPRECATED_62):
```

Comment:
I understood it was bad to remove ops?

Reply:
Ah, I see, the _u candidates are using previously deprecated slots.  Won't that affect being able to execute older binaries?

Reply:
You mean old precomp files? I believe it's been done before, the reason being that it means the moarvm files don't increase in size quite as much as if we always added new ops in new slots. However, as I'm typing this I realized that I just used the next deprecated slot down the oplist, but maybe I should instead have used the oldest? Ping @niner @jnthn @timo

Reply:
Deprecated here means "any use of the op will throw an exception", not "you can use it but should stop soon". So anything still referencing them will have long been broken.